docker/docker_exec.go

```diff
@@ -155,7 +155,10 @@ func (sa *stdioAttacher) withRawTerminal(fn attachFn) <-chan error {
 		// characters
 		prevState, err = term.SetRawTerminal(fd)
 		if err != nil {
-			done <- err
+			go func() {
```

Comment:
maybe it's better to make the channel buffered?

Reply:
Even if in this case would slightly increase readability, I think that using an unbuffered channel keeps the code simpler compared to the readability gain of using a buffered channel.

Reply:
what exactly do you see as a downside of a buffered channel?

Reply:
it's just an easier model to think 😄